### PR TITLE
EVD imei provisioning (added details)

### PR DESCRIPTION
**Pull request by shivani61**

### CODE REVIEW

File: docs/home/devconsole/device-template/imei-provisioning-template/index.md [hunk 1]
```diff
@@ -7,17 +7,43 @@ IMEI or Serial number of devices to be provisioned need to be added to the relev
...
+![](../../images/template_for_imei4.png)
+
+Then on the resulting dialog expand the IMEI/Serial section, and then you can see the serial numbers associated with the template.
+
+![](../../images/template_for_imei5.png)
+
+7. To delete a serial number from a template, simply follow steps 1 to 4, and click on the trash-can icon to delete a serial number from the template.
+
+
+All this means that during manufacturing, we will need to get a list of serial numbers for all Kiosks, and the serial number should ideally be listed somewhere on the unit, so that it can be easily identified. Serial numbers can be added in bulk using CSV files.
```
Comment (bmehendale) [inline docs/home/devconsole/device-template/imei-provisioning-template/index.md]: Please remove this sentence -- it was meant for OEM manufacturing scenario.

File: docs/home/devconsole/device-template/imei-provisioning-template/index.md [hunk 2]
```diff
@@ -7,17 +7,43 @@ IMEI or Serial number of devices to be provisioned need to be added to the relev
...
+![](../../images/temp_2.png)
 
-During editing, you also see the option to add IMEI/Serial numbers of devices you want to provision with this template. There is a section "Add devices" that appears during edit just before Preview section. Using this "Add devices" section, user can either enter IMEI/Serial numbers or upload Csv file with IMEI and Serial numbers.
+2. On the page that pops up, click on Add devices:
 
-The user can enter IMEI numbers and Serial numbers in the text box and click Add.
+![](../../images/template_for_imei1.png)
 
-OR
+3. Then grab the serial number of the device which will be seamlessly provisioned, and enter it as shown. Then click on the Add button:
```
Comment (bmehendale) [inline docs/home/devconsole/device-template/imei-provisioning-template/index.md]: Please expand the process of acquiring the serial number from the AVD. i.e. either using ADB or using the one that's displayed on the QR code screen reader.

File: docs/home/devconsole/device-template/imei-provisioning-template/index.md (hunk 1 above)
Comment (shivani61) [reply docs/home/devconsole/device-template/imei-provisioning-template/index.md]: @bmehendale  i already removed it i think.

File: docs/home/devconsole/device-template/imei-provisioning-template/index.md (hunk 2 above)
Comment (shivani61) [reply docs/home/devconsole/device-template/imei-provisioning-template/index.md]: added the same

**Pull request merged**